### ISSUE TITLE
meeting.appShareButton.setOptions() not working

### ISSUE THREAD

**Issue by julianbrasolin** [OfficeDev/microsoft-teams-library-js#1489]
Hi, im calling this method with a boolean parameter but i cant seem to get it working. it does not throw any errors or anything, it just won't do anything, not show nor hide the app sharing (start presenting) button.

`let newShareInformation: microsoftTeams.meeting.appShareButton.ShareInformation = {
            isVisible: true,
 };
 microsoftTeams.meeting.appShareButton.setOptions(newShareInformation);`

**Comment by sayali-MSFT** [comment]
@julianbrasolin - Thanks for reporting your issue.
We will check the method from our end and get back to you.

**Comment by sayali-MSFT** [comment]
@julianbrasolin -Currently Engineering team working on this to fix this issue, we will let you know once we have any updates on it. 
Thanks.

**Comment by ryanbliss** [comment]
@julianbrasolin Note that this is an `@beta` API and is not officially supported in the Teams client yet (or at least not in all environments). I'm surprised this made it into --latest version of teams-js. The docs say that the default setting is `isVisible: false`, but in reality the current default setting is `isVisible: true`. Something to follow up on @sayali-MSFT

**Comment by ChetanSharma-msft** [comment]
Hello @julianbrasolin,
Hope you are doing well!!

Just wanted to inform you that this issue has been converted to a feature request by engineering team.
Currently we don't have exact ETA to share but will let you know once we get any update from engineering team.

**Comment by nwojod-MSFT** [comment]
Hello @julianbrasolin, apologies for the delayed update on this issue. This API is still currently being developed by the team and not available to be used yet. We will notify everyone once this is ready. Thanks.
